shared/src/metabase/shared/util/currency.cljc

```diff
@@ -873,9 +873,19 @@
           :code "ZMK",
           :name_plural "Zambian kwachas"}]])
 
+(def currency-map
+  "The currencies as a Clojure map. Wrapped in [[delay]] so it's only computed on demand."
```

Comment:
Why is it important to compute it only on demand?

Reply:
I dunno, but it's easy to use `delay` to preserve the caching logic in the [original](https://github.com/metabase/metabase/blob/f5d85e8131903976d4b4194f7f5de728b522fe9c/frontend/src/metabase/lib/formatting/currency.js#L8).